### ISSUE TITLE
Running this ansible script causes Hyper terminal emulator to break

### ISSUE THREAD

**Issue by jcklpe** [ReinerNippes/nextcloud#61]
I have no clue why this is happening but I've been able to replicate it three times, with two different distros (First with my old Ubuntu installation, the second time when I tried reinstalling Linux and nextcloud fresh with PopOS and then the third time with a completely fresh Ubuntu installation). 

Prior to running the ansible script, Hyper works as expected. As soon as I run this script, Hyper no longer works. 

This is frankly very bizarre. 

I'm going to try and manually install Nextcloud and see if that fixes the issue.

**Comment by jcklpe** [comment]
@ReinerNippes A friend of mine who knows a lot more Linux than me came by and figured out the problem! 

So the issue is that the prep_os ansible task locks tmpfs from executing, which is a problem because Hyper, using node-pty, tries to write a chromium process to tmp but it can't because the permissions have been changed. 

Why does this happen? And can this perhaps be changed so that other people don't have the same issue? 

My friend fixed the issue on my system by creating a tmp2 folder for hyper to write to but seems like this could probably cause problems for other people etc.

**Comment by ReinerNippes** [comment]
This behaviour is removed in the branch "nextcloud-reloaded". 
If you setup nextcloud everything is fine and no need for you to change anything. 
If you want to start over again, do a `git checkout nextcloud-reloaded` before you edit the inventory file. 

Nevertheless I changed so much between this two versions of the playbook that they aren't compatible. So don't run the new version on a system that was setup with the old.

**Comment by jcklpe** [comment]
@ReinerNippes Thanks, will do! Also I encountered a problem with OnlyOffice. This is a fresh install like I said, so not sure what's up. Basically I try to open a document and I get an error 

![image](https://user-images.githubusercontent.com/9451726/70482172-66391180-1aaa-11ea-8ad1-d44e8a5ebcf2.png)
I'm currently going through old issues to see if this can be fixed by something we previously discussed but so far haven't found the issue.

**Comment by ReinerNippes** [comment]
This looks like the error I got when using self-signed certificates for nextcloud. 
If that's the case you'll find the solution here: https://github.com/ONLYOFFICE/Docker-DocumentServer/issues/96#issuecomment-552626862

**Comment by jcklpe** [comment]
@ReinerNippes  

Is this the solution you were referring to? `docker exec -it containerID /var/www/onlyoffice/documentserver/npm/node_modules/.bin/json -f /etc/onlyoffice/documentserver/default.json -I -e 'this.services.CoAuthoring.requestDefaults.rejectUnauthorized=false'` ?

**Comment by ReinerNippes** [comment]
yes. but keep in mind it's only in the container set to false and won't survive an update.

**Comment by jcklpe** [comment]
Excellent, that worked. thanks. I will keep my eye out for the next version. I didn't end up reinstalling "nextcloud-reloaded" but I I have a feeling that it won't be many more months before I'm forced to redo my installation all over again. 

I wish I could help you more instead of always asking you for help all the time. If there's anything I can do, let me know. I've been doing this "running my own nextcloud" thing for nearly a year and I still feel like I am pretty clueless on how a lot of stuff works.

 I was having so many problems with my installation that I decided to completely reinstall both linux and nextcloud and that's fixed things, but I 

1. realized how little I understood stuff the first time I installed Linux/Nextcloud (this time around I understood it much better and when I googled a fix for something I actually understood a little more of what I was actually doing)
2. I see how much more now that I don't really understand what I'm doing D: 

I'm going to write a note here for my own documentation purposes so that next time I reinstall Nextcloud and linux I remember:

- disabled restic (first time that lead to me overfilling my 250gb SSD since I'm not running an automatic backup protocol (IE I'm not properly doing a real cloud set up. I'm self hosting this out of my closet))
- disable fail2ban
-
